app/src/main/java/com/example/androidseminar/activity/OnboardingActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.androidseminar.activity
+
+import android.os.Bundle
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.ui.setupActionBarWithNavController
+import androidx.room.Room
+import com.example.androidseminar.R
+import com.example.androidseminar.databinding.ActivityOnboardingBinding
+import com.example.androidseminar.util.BaseActivity
+
+class OnboardingActivity : BaseActivity<ActivityOnboardingBinding>({ ActivityOnboardingBinding.inflate(it)}) {
```

Comment:
```suggestion
class OnboardingActivity : BaseActivity<ActivityOnboardingBinding>(ActivityOnboardingBinding::inflate) {

// or
class OnboardingActivity : BaseActivity<ActivityOnboardingBinding> { ActivityOnboardingBinding.inflate(it) } {
```

마지막에 오는 람다이기에 괄호안에 함수는 안적으셔도 됩니다. 아니면 위에처럼 직접 함수를 참조하셔서 넣으셔도 되구요